src/components/accordion/Accordion.jsx

```diff
@@ -115,36 +204,28 @@ const Accordion = ({
     };
   }, [state.focusedElementId]);
 
-  function getUpdatedOpenedItems(
-    opened: OpenedItemsType,
-    id: string,
-    value: boolean
-  ) {
-    return allowMultiple ? {...opened, [id]: value} : {[id]: value};
-  }
-
   function reducer(state: StateType, action: ActionType): StateType {
     switch (action.type) {
-      case 'accordion/SET_OPENED': {
-        const {id, value} = action.payload;
+      case 'accordion/SET_EXPANDED': {
+        const {expanded} = action.payload;
 
         return {
           ...state,
-          opened: getUpdatedOpenedItems(state.opened, id, value),
+          expanded,
```

Comment:
don't we need `getUpdatedOpenedItems` ? what about the case when `allowMultiple = true` ? 🤔

Reply:
we don't! resolved ✅